oauth2client/flask.py

```diff
@@ -0,0 +1,299 @@
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utilities for the Flask web framework
+"""
+from __future__ import absolute_import
+import json
+import random
+import string
+from functools import wraps
+
+import httplib2
+from flask import (Blueprint, current_app, redirect, request, session, url_for,
```

Comment:
1. Will do, this came from a project that absolutely had Flask.
2. Will do, see my note about style in the PR message.
3. Will run the importer check on it as well.